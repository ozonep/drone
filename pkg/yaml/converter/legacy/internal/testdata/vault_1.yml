pipeline:
  docker:
    image: plugins/docker
    secrets: [ docker_username, docker_password ]
    repo: octocat/hello-world

secrets:
  docker_username:
    driver: vault
    driver_opts:
      path: secret/docker/username
  docker_password:
    driver: vault
    driver_opts:
      path: secret/docker
      key: password
