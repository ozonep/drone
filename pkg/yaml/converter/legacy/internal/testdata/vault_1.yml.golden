---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: docker
  pull: if-not-exists
  image: plugins/docker
  settings:
    repo: octocat/hello-world
  environment:
    DOCKER_PASSWORD:
      from_secret: docker_password
    DOCKER_USERNAME:
      from_secret: docker_username

---
kind: secret
name: docker_password

get:
  path: secret/docker
  name: password

---
kind: secret
name: docker_username

get:
  path: secret/docker/username

...
